introduction.md

```diff
@@ -1,3 +1,4 @@
 This is an introduction
 
-JavaScript
\ No newline at end of file
+*JavaScript
+*Git
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?